src/app/tvshows/tvshows-search.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { TvDbService } from '../services';
+import { NgRedux, select } from '@angular-redux/store';
+import { RootState } from '../store';
+import { searchShows } from './store/tvshows.actions';
+import { TvShowData } from './store/tvshows.models';
+
+@Component({
+  selector: 'app-shows-search',
+  template: `
+  <form action="javascript:void(0)">
+    <input type="text" (change)="search($event.target.value)" placeholder="Show Title">
+    <input type="submit">
+  </form>
+  <div *ngIf="isFetching$ | async">
+    <progress value="80" max="100">fetching...</progress>
+  </div>
+  <ul>
+    <li *ngFor="let show of (tvshows$ | async)">{{ show.title }}</li>
+  </ul>
+  `
+})
+export class TvShowsSearchViewComponent {
+ @select(['tvshows', 'searchResults']) readonly tvshows$: Observable<TvShowData[]>;
```

Comment:
Looks like your indentation is off in here (one space versus two)

Reply:
Yes, I changed the linter config to ensure this is checked.